lib/slimmer/app.rb

```diff
@@ -28,10 +28,7 @@ def initialize(app, *args, &block)
         options[:asset_host] = Plek.current.find("static")
       end
 
-      cache = Cache.instance
-      cache.use_cache = options[:use_cache] if options[:use_cache]
```

Comment:
I might be wrong but this looks like the cause of the cache not be working, I'm not sure where `options[:use_cache]` is coming from but it's likely that it got forgotten so the default (false) will be use (https://github.com/alphagov/slimmer/pull/183/files#diff-3edabcdd528cf7a4390669a093f8f185L12)